docs/onboarding/cli.md

```diff
@@ -253,6 +256,14 @@ ironfish faucet
 ```
 
 ### Chain
+
+#### chain:asset
+Get the asset info by asset identifier
+
+```sh
+ironfish chain:asset <identifier of an existing asset>
```

Comment:
can you have dash to keep the consistency of the doc, so <identifier-of-an-existing-asset>